### PR TITLE
flutter update-packages

### PR DESCRIPTION
**Pull request by christopherfujino**
Updates `package:dds` to 2.1.1 which should address https://github.com/flutter/flutter/issues/86361

### CODE REVIEW

File: dev/bots/pubspec.yaml [hunk 1]
```diff
@@ -16,8 +16,9 @@ dependencies:
   process: 4.2.3
   test: 1.17.11
 
-  _fe_analyzer_shared: 23.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  analyzer: 2.0.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  _discoveryapis_commons: 1.0.1 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```
Comment (christopherfujino) [inline dev/bots/pubspec.yaml:19]: it looks like these additional deps were actually added to dev/devicelab in https://github.com/flutter/flutter/pull/88447/files

**Review by bkonyi — APPROVED** [review]
LGTM!

**Comment by zanderso** [comment]
This is updating way more than just DDS. It updates the analyzer, frontend_server_client, and vm_service. In the past, for crashes in DDS, I thought we had been able to update to point releases of DDS containing only the needed fix, and not any additional package version updates. That what I was expecting would be done here as well.

**Comment by zanderso** [comment]
Oh, I see. The cherry-pick PR does only update to the point release. Do I have that right?

**Comment by christopherfujino** [comment]
> Oh, I see. The cherry-pick PR does only update to the point release. Do I have that right?

Correct. Sorry, this was confusing, as our release process requires an issue, and assumes that the commit fixing this issue will be cherrypicked.

What I actually did was manually just increment DDS to the point release with just the fix isolated, that is 2.0.2 -> 2.0.3

**Pull request merged**